md/release_notes.md

```diff
@@ -1,17 +1,31 @@
 # Release notes
 
-::: info
-**Note:** The 1.3 is currently work in progress (WIP).
-:::
+## New values
+### Task assignment decision support 
+
+When a case seems stuck or gets pretty late, view all candidates for the pending task and then reassign the task to the person with the lesser load, or you know will be more available to do the task and get the case back on track.
```

Comment:
```suggestion
When a case seems stuck or gets pretty late, view all candidates for the pending task and then reassign the task to the correct person. Choose the candidate with the smaller workload or one that you expect to be more available to do the task and get the case back on track.
```

Reply:
This suggestion won't be integrated, @NathalieC already change the sentence with an alternative wording.
So mark as resolved